ffmpeg/extras.c

```diff
@@ -176,6 +176,10 @@ int lpms_get_codec_info(char *fname, pcodec_info out)
       }
       out->width  = ic->streams[vstream]->codecpar->width;
       out->height = ic->streams[vstream]->codecpar->height;
+      
+      AVRational frame_rate = av_guess_frame_rate(ic,ic->streams[vstream],NULL);
+      out->fps = (frame_rate.num && frame_rate.den ? av_q2d(frame_rate) : 0);
```

Comment:
What does this line do? Why isn't it just this?
```
out->fps = frame_rate.num;
```

Reply:
frame rates are in `AVRational` types that are similar to Rats in Go with a numerator and denominator.  I think this is overly conservative after researching a bit more and changing to `avg_frame_rate` in the AVStream